lattice_mc/simulation.py

```diff
@@ -271,23 +271,25 @@ def collective_correlation( self ):
             return None
 
     @property
-    def reduced_ionic_conductivity( self ):
+    def collective_diffusion_coefficient( self ):
         """
-        Returns the reduced ionic conductivity, \u03C3\' ( D_J * n_natoms).
+        Returns the collective or "jump" diffusion coefficient multiplied by the number of atoms,
+        ( D_J * number_of_atoms).
 
         Args:
             None
 
         Returns:
-            (Float): The reduced ionic conductivity, \u03C3\' ( D_J * n_natoms).
+            (Float): the collective or "jump" diffusion coefficient multiplied by the number of
+            atoms, ( D_J * number_of_atoms).
         """
         if self.has_run:
             return self.atoms.collective_dr_squared() / ( 6.0 * self.lattice.time )
         else:
             return None
 
     @property
-    def collective_diffusion_coefficient( self ):
+    def collective_diffusion_coefficient_per_atom( self ):
         """
         Returns the collective or "jump" diffusion coefficient, D_J.
```

Comment:
This is incorrect. Returns the collective diffusion coefficient per atom, which is equal to the "jump" diffusion coefficient.